### PR TITLE
Feat: [OR-153] Listbox rebrand

### PR DESCRIPTION
**Pull request by vicky-comeau**
## Summary

Rebranded the Listbox component

### CODE REVIEW

File: packages/components/src/listbox/src/Listbox.css [hunk 1]
```diff
@@ -62,123 +82,136 @@ IMPORTANT: The Listbox component hardcoded a few CSS values to enable dynamic sc
...
-}
-
-/* AVATAR */
-.o-ui-listbox-option-avatar {
-    margin-right: var(--o-ui-sp-2);
-    grid-area: aside;
+    color: var(--hop-danger-icon-active);
+}
+
+.o-ui-listbox-invalid .o-ui-listbox-option[aria-selected="true"]:not([aria-disabled="true"]):not(.o-ui-listbox-option-no-hover):hover,
```
Comment (fraincs) [inline packages/components/src/listbox/src/Listbox.css:180]: These could most likely be reworked to prevent too many declarations.

**Comment by vicky-comeau** [comment]
> image image In some instances (long words) the font weight change is enough to make list width change by 1px like with Neptune.

Is this really an issue? the width is set to max-content. A font-weight change is bound to take up more width.

File: packages/components/src/listbox/src/Listbox.tsx [hunk 2]
```diff
@@ -365,8 +365,10 @@ export function InnerListbox({
     });
 
     const scrollableProps = useScrollableCollection(containerRef, nodes, {
+        borderHeight: 2,
```
Comment (fraincs) [inline packages/components/src/listbox/src/Listbox.tsx:368]: how are these used?

Comment (vicky-comeau) [reply packages/components/src/listbox/src/Listbox.tsx:368]: The new design adds a border and padding to the list, which the previous one didn't have so we need to add then when calculating the scroll height.

**Comment by fraincs** [comment]
> the width is set to max-content

But on load max-content will be 1px less.

**Comment by fraincs** [comment]
image

Icon and text should share the same color, set the color on the li level and you should be good.

**Review by fraincs — APPROVED** [review]

**Review by fraincs — APPROVED** [review]

**Pull request merged**